### PR TITLE
Update curalate-ops-cli to 0.0.19

### PR DESCRIPTION
**Pull request by timothyhahn**
Includes fixes from https://github.com/curalate/curalate-ops-cli/pull/3.

@bchen421, mind taking a look?

### CODE REVIEW

File: curalate-ops-cli.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@ class CuralateOpsCli < Formula
 
   desc "Curalate Ops Jobs CLI"
   homepage "https://github.com/curalate/curalate-ops-cli"
-  url "https://github.com/curalate/curalate-ops-cli/archive/0.0.18.tar.gz"
-  version "0.0.18"
-  sha256 "741457ac986f96aa229f30bed35a62b275bbaf0b0b63a3af585ebb7bddba5f69"
+  url "https://github.com/curalate/curalate-ops-cli/archive/0.0.19.tar.gz"
+  version "0.0.19"
+  sha256 "872fd30e8e95c358001e27642f826ef85fc33f2611709229a7e7c8f60a189dc2"
```
Comment (bchen421) [inline curalate-ops-cli.rb]: Just curious what you ran to get the sha256.  I get the following:
```
shasum -a 256 ~/Downloads/curalate-ops-cli-0.0.19.tar.gz
9c64b2d53340b006fd2e781db7cbf6f0e4f4d1c3400c43e874930a29f9182dba  /Users/bchen/Downloads/curalate-ops-cli-0.0.19.tar.gz```

Comment (timothyhahn) [inline curalate-ops-cli.rb]: Hmm, I ran a script I wrote a while back:

```
curl -L "https://github.com/curalate/curalate-ops-cli/archive/$VERSION_TAG.tar.gz" -o $FILE_NAME

echo "Here is the sha 256 sum you need to update https://github.com/curalate/homebrew-tap/blob/master/curalate-ops-cli.rb"
shasum -a 256 $FILE_NAME
```

Lemme try this again.

Comment (timothyhahn) [inline curalate-ops-cli.rb]: Uhm....

```
 韓  ✘ tim@lain  /tmp  shasum -a 256 0.0.19.tar.gz
872fd30e8e95c358001e27642f826ef85fc33f2611709229a7e7c8f60a189dc2  0.0.19.tar.gz

 韓  ✘ tim@lain  /tmp  curl -L https://github.com/curalate/curalate-ops-cli/archive/0.0.19.tar.gz -o test.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   184    0   184    0     0    771      0 --:--:-- --:--:-- --:--:--   769
100 7011k  100 7011k    0     0   652k      0  0:00:10  0:00:10 --:--:--  796k

 韓  tim@lain  /tmp  shasum -a 256 test.tar.gz
9c64b2d53340b006fd2e781db7cbf6f0e4f4d1c3400c43e874930a29f9182dba  test.tar.gz
 韓  tim@lain  /tmp  shasum -a 256 0.0.19.tar.gz
9c64b2d53340b006fd2e781db7cbf6f0e4f4d1c3400c43e874930a29f9182dba  0.0.19.tar.gz
```

WTF? I'mma go with the other SHA, but this is really weird.

**Comment by bchen421** [comment]
Weird

**Pull request merged**